src/queries/metadata.ts

```diff
@@ -0,0 +1,52 @@
+import { gql } from '@apollo/client';
+
+export const METADATA = gql`
+    query {
+        allMetadata {
+            id
+            name
+            description
+            published
+            source
+            updated
+            tags
+            visualisations {
+                axes {
+                    x {
+                        name
+                    }
+                    y {
+                        name
+                    }
+                }
+            }
+        }
+    }
+`;
+
+export type MetadataEntry = {
+    id: string;
+    name: string;
+    description: string;
+    published: string;
+    source: string;
+    updated: string;
+    tags: string[];
+    visualisations: [
+        {
+            type: string;
+            axes: {
+                x: {
+                    name: string;
+                };
+                y: {
+                    name: string;
+                };
+            };
+        },
+    ];
+};
+
+export type Metadata = {
```

Comment:
This seems a bit strange. Metadata to me sounds more like what `allMetadata` returns an array of.

Could you perhaps give it a better name? What about just `AllMetadataResult` or something along those lines?